.ci/jenkins/Jenkinsfile.tools.update-quarkus

```
@@ -0,0 +1,56 @@
+import org.jenkinsci.plugins.workflow.libs.Library
+
+@Library('jenkins-pipeline-shared-libraries')_
+
+REPOS = [ 'kogito-runtimes', 'kogito-examples', 'optaplanner', 'optaplanner-quickstarts' ]
+
+pipeline {
+    agent {
+        label 'kie-rhel7 && !master'
+    }
+
+    // parameters {
+    // For parameters, check into ./dsl/jobs.groovy file
+    // }
+
+    // environment {
+    // Some generated env is also defined into ./dsl/jobs.groovy file
+    // }
+
+    stages {
+        stage('Initialize') {
+            steps {
+                script {
+                    assert getNewVersion()
+
+                    currentBuild.displayName = getNewVersion()
+                }
+            }
+        }
+
+        stage('Call update quarkus jobs') {
+            steps {
+                script {
+                    REPOS.each { repo ->
```

`REPOS` is not used again, but I guess you follow that approach just to have the list of repositories more clear, isn't it? So up to you whether to commit this suggestion
```suggestion
                    [ 'kogito-runtimes', 'kogito-examples', 'optaplanner', 'optaplanner-quickstarts' ].each { repo ->
```

---

updated